wastewater/dataModel.WasteWaterTreatment/WasteWaterTreatment-schema.json

```diff
@@ -1,80 +1,4 @@
 {

```

Comment:
missing the header of this file

Reply:
Hi Alberto,

Thanks I have added the header of the file. I have a question regarding the base schema. What properties are we needed to define in the base schema? From all the examples, it is not clear to me. The only one that I can understand is the Commons definitions.

Until now I was under the impression, we need to define all possible properties of the various entities within the wastewater data models (Tank, Blower, OffGasStack etc.) in the base schema as well. But from the examples, that does not seem to be the case?

Reply:
The properties have to be defined only in one place (either at common-schema.json, in a shared schema like [this](https://github.com/smart-data-models/dataModel.WaterNetworkManagement/blob/master/WaterNetworkManagement-schema.json) and then call through a $ref or directly in the data model).
In the case that they are defined in two places this would be an error. If you find any please let me know.